frontend/doc-site/annotating-data.mdx

```diff
@@ -0,0 +1,117 @@
+
+#### 3. Merging draft annotations with the main h5ad file
+
+Once you're finished with your annotations, you should finalize and preserve your work by merging your `csv` into your main `h5ad` file.
+
+You can do so like this:
+
+```text
+import pandas as pd
+import scanpy as sc
+
+new_annotations = pd.read_csv('myannotations.csv',
+                         comment='#',
+                         dtype='category',
+                         index_col=0)
+anndata = sc.read('mydata.h5ad')
+anndata.obs = anndata.obs.join(new_annotations)
+```
+
+## Annotations by multiple users
+
+As described in the [hosted](https://github.com/chanzuckerberg/cellxgene/blob/main/docs/posts/hosted) section, we do not officially support hosted or multi-user use of cellxgene. However, we recognize that the app is often adapted for this purpose, and have tried to provide a "safe path" for multi-user setups that avoids overwriting data.
+
+Specifying a single file name for multiple contributors will result in data overwriting. To avoid this, you can instead specify an output directory and allow cellxgene to assign filenames.
+
+To specify an output directory, run:
+
+```text
+cellxgene launch mydata.h5ad --annotations-dir path/to/annotations-directory/
+```
+
+For each user, annotations will be saved as follows:
+
+* Each user will be prompted to enter a name for their annotations the first time they create a new category.
+* We also assign a unique identifier in the form of an 8-character suffix, `########`; this helps cellxgene identify their specific file to avoid overwriting others' work.
+* Any annotations created in the application will be autosaved in `annotations-directory/name-########.csv`
+
+### FAQ
+
+#### How do I know my annotations are saved?
+
+`cellxgene` autosaves any changes made to your annotations every 3 seconds.
+
+#### I think I deleted my annotations! Oh noes!
+
+Not to worry! We save the last 10 versions of your annotations in `annotations-directory/NAME-backups/`
+
+#### What about creating continuous annotations?
+
+Continuous metadata is important! However, these values \(e.g., pseudotime\) are the result of statistical analyses that are beyond cellxgene's visualization- and exploration-focused scope. We do, of course, provide visualization of continuous metadata values computed elsewhere and stored in `anndata.obs`.
+
+#### I keep getting weird index errors when trying to join my annotations to my anndata?
+
+This is most likely because the h5ad file you are working with is not the original file used to generate the annotations! We recommend merging new annotations in on a regular basis for this reason.
+
+#### How do you remember my unique ID to match my cellxgene session with my annotations file?
+
+We place a small cookie \(file\) in your browser that identifies where your draft annotations are saved. This file never leaves your machine, and is never sent to the cellxgene team or anyone else.
```

Comment:
how often is it updated? if someone starts a session and comes back to it a few weeks later will it get a new id?